aday2dream/src/main/kotlin/com/aday2dream/aday2dream/controller/AccountController.kt

```diff
@@ -0,0 +1,128 @@
+package com.aday2dream.aday2dream.controller
+
+import com.aday2dream.aday2dream.dto.AccountDto
+import com.aday2dream.aday2dream.dto.AccountLoginDto
+import com.aday2dream.aday2dream.dto.AccountRegisterDto
+import com.aday2dream.aday2dream.entity.Account
+import com.aday2dream.aday2dream.service.AccountService
+import com.aday2dream.aday2dream.service.JwtBlacklistService
+import com.aday2dream.aday2dream.service.JwtService
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.security.authentication.AuthenticationManager
+import org.springframework.security.authentication.BadCredentialsException
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.security.core.userdetails.UserDetails
+import org.springframework.security.core.userdetails.UsernameNotFoundException
+import org.springframework.security.web.authentication.logout.SecurityContextLogoutHandler
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/accounts")
+class AccountController(@Autowired private val accountService: AccountService,
```

Comment:
I am leaving one comment for the whole request so to not repeat it many times but:
1) add logging using Slf4j to enhance the monitoring and debugging of your app
2) use @AllArgsConstructor in Controllers and Services (Components in general) when you need dependency injection
3) Unit tests are missing, you should test your Controllers and Services!

Reply:
Also, remove the .idea files!!!